test/integration/network_ips_test.go

```diff
@@ -43,6 +44,33 @@ func TestIPAddress_GetFound_smoke(t *testing.T) {
 	if i.Address != address {
 		t.Errorf("Expected a specific ipaddress, but got a different one %v", i)
 	}
+
+	// Test for fetching reserved IP
+	// First, create a reserved IP
+	reservedIP, err := client.ReserveIPAddress(context.Background(), linodego.ReserveIPOptions{
```

Comment:
Just a heads up, I'm getting 500s when trying to allocate reserved IPs in `ap-west` (where this test's instance is provisioned